BESMod/Systems/Demand/Building/SpawnHighOrder.mo

```diff
@@ -0,0 +1,238 @@
+  parameter Integer nZonesNonHeated = 1 "Non heated rooms of the building";
+  parameter Integer nZones = 10 "Heated rooms of the building";
+  parameter Boolean useConstVentRate=false;
+  parameter Real ventRate[nZones]=fill(0, nZones) if useConstVentRate "Constant mechanical ventilation rate" annotation (Dialog(enable=useConstVentRate));
+  parameter String idf_name=Modelica.Utilities.Files.loadResource("modelica://BESMod/Resources/Spawn/AixLib_HOM_in_EnergyPlus.idf")        "Name of the IDF file";
+  parameter String epw_name=Modelica.Utilities.Files.loadResource("modelica://BESMod/Resources/Spawn/Potsdam_TRY2015_normal.epw")        "Name of the weather file, in .epw format and with .epw extension";
+  parameter String wea_name=Modelica.Utilities.Files.loadResource("modelica://BESMod/Resources/Spawn/Potsdam_TRY2015_normal.mos")        "Name of the weather file, in .mos format and with .mos extension";
+  inner Buildings.ThermalZones.EnergyPlus_9_6_0.Building building(
+    idfName=idf_name,
+    epwName=epw_name,
+    weaName=wea_name)
+    annotation (Placement(transformation(extent={{-94,-10},{-74,10}})));
+  Components.SpawnHighOrderOFD.GroundFloor groundFloor(redeclare package Medium =
+        MediumZone, VZones=VZone[1:5])
+    annotation (Placement(transformation(extent={{-24,-88},{32,-40}})));
+  Components.SpawnHighOrderOFD.UpperFloor upperFloor(redeclare package Medium =
+        MediumZone, VZones=VZone[6:10])
+    annotation (Placement(transformation(extent={{-24,-20},{30,28}})));
+  Modelica.Blocks.Math.Add TZoneOpeMeaKelvin[nZones + nZonesNonHeated](each k1=0.5,
+      each k2=0.5) "Operative room temperature"
+    annotation (Placement(transformation(extent={{52,-8},{66,6}})));
+  Components.SpawnHighOrderOFD.Attic attic_unheated(redeclare package Medium =
+        MediumZone, VZone=VZone[11])
+    annotation (Placement(transformation(extent={{-14,36},{18,66}})));
+  Modelica.Blocks.Sources.Constant constVenRatAtt(final k=1)
+    "Constant ventilation rate of attic"
+    annotation (Placement(transformation(extent={{-86,36},{-70,52}})));
+  Modelica.Blocks.Math.Gain InternalGainsConvRatio[nZones](each k=0.6)
+    annotation (Placement(transformation(extent={{80,-58},{70,-48}})));
+  Modelica.Blocks.Math.Gain InternalGainsRadRatio[nZones](each k=0.4)
+    annotation (Placement(transformation(extent={{80,-76},{70,-66}})));
+  Utilities.Electrical.ZeroLoad zeroLoad
+    annotation (Placement(transformation(extent={{30,-108},{50,-88}})));
+  Modelica.Blocks.Sources.Constant constVentRate[nZones](final k=ventRate)
+    if useConstVentRate "Only used when \"useConstVentRate = true\""
+                                      annotation (Placement(transformation(
+          extent={{10,-10},{-10,10}}, rotation=180,
+        origin={-84,-30})));
+  Modelica.Thermal.HeatTransfer.Celsius.FromKelvin TZoneOpeMea[nZones +
+    nZonesNonHeated]
+    annotation (Placement(transformation(extent={{60,22},{68,30}})));
```

Comment:
All units are always in SI in BESMod, so the bus output should be in Kelvin, as well.